values.yaml

```diff
@@ -73,7 +73,12 @@ beyla:
       memory: 1536Mi
     requests:
       cpu: 250m
```

Comment:
@paweljw Could you also make the CPU requests equal to CPU limits to get into Guaranteed QoS category and be last to get killed? 🙏

Reply:
And actually, the collector container should also have requests == limits imho.